staker/block_validator.go

```diff
@@ -457,18 +471,18 @@ func (v *BlockValidator) sendRecord(s *validationStatus) error {
 			log.Error("Error while recording", "err", err, "status", s.getStatus())
 			return
 		}
+		validatorValidationRecordingHist.Update(s.timeStampInterval())
 		if !s.replaceStatus(RecordSent, Prepared) {
 			log.Error("Fault trying to update validation with recording", "entry", s.Entry, "status", s.getStatus())
 			return
 		}
-		nonBlockingTrigger(v.progressValidationsChan)
```

Comment:
Why was this removed? Shouldn't advanceValidations be called after setting the status to prepared to launch the validation run?

Reply:
I.. don't know how it got removed